benchmarks/run_ep_matrix_one_pt.sh

```
@@ -0,0 +1,37 @@
+WHEEL_EP=$(python -c "import nccl.ep,os;print(os.path.dirname(nccl.ep.__file__))")
+NCCL_LIB=$(python -c "import nvidia.nccl,os;print(list(nvidia.nccl.__path__)[0]+'/lib')")
+NCCL_INC=$(python -c "import nvidia.nccl,os;print(list(nvidia.nccl.__path__)[0]+'/include')")
+export LD_LIBRARY_PATH="$NCCL_LIB:$WHEEL_EP/lib:/usr/local/cuda/lib64:${LD_LIBRARY_PATH:-}"
+
+# HT runtime JIT: use the base image's complete CUDA toolkit (>=13.2, matching
+# NCCL's +cuda13.3 build). nccl_device.h ships in the nvidia-nccl wheel include,
+# reached via CPATH (nvcc honors it) — no combined symlink dir needed.
+export NCCL_EP_JIT_SOURCE_DIR="$WHEEL_EP/include/nccl_ep"
+export NCCL_EP_JIT_BUILD_INCLUDE_DIR="$WHEEL_EP/include"
+export NCCL_EP_JIT_CUDA_INCLUDE_DIR="${CUDA_HOME:-/usr/local/cuda}/include"
+export NCCL_EP_JIT_NVCC="$(command -v nvcc)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)